articles/fleet-4.15.0.md

```diff
@@ -0,0 +1,77 @@
+# Fleet 4.15.0 adds beta support for self-service, increases transparency, and brings Zendesk to the party.
+
+![Fleet 4.15.0](../website/assets/images/articles/fleet-4.15.0-cover-1600x900@2x.jpg)
+
+Fleet 4.15.0 is here. Check out the full [changelog](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.15.0) or continue reading to get the highlights.
+
+For update instructions, see our [upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs.
+
+## Highlights
+- Self-service with Fleet Desktop. 
+- Transparency with Fleet Desktop.
+- Zendesk integration
+- "Last used" information for macOS applications.
+
+## Self-service with Fleet Desktop on macOS, Windows, and Linux
+**Available in Fleet Premium**
+
+![Self-service with Fleet Desktop on macOS, Windows, and Linux](../website/assets/images/articles/fleet-4.15.0-policy-self-service-1600x900@2x.jpg)
+Fleet Desktop is a lightweight application that allows end-users to self serve failing policies on their laptop or workstation.
+
+With Fleet Desktop installed on a host, the end-user has access to the list of policies that apply to their laptop, and the instructions to resolve failing policies. The end-user can quickly reach this list of policies by selecting the Fleet icon > My device in the menu bar.
```

Comment:
```suggestion
With Fleet Desktop installed on a host, the end-user has access to the list of policies that apply to their laptop, and the instructions to resolve failing policies. The end-user can quickly reach this list of policies by selecting the **Fleet icon > My device** in the menu bar.
```

I suggest adding bold styles to indicate that this is the title of a UI element (button).